sql/moz-fx-data-shared-prod/search_terms_derived/sanitation_job_metadata/init.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.search_terms_derived.sanitation_job_metadata`
+AS
+SELECT
+  *
+FROM
+  `mozdata.search_terms_unsanitized_analysis.prototype_sanitation_job_metadata`
+WHERE
+  DATE(timestamp) >= (CURRENT_DATE() - 14)
```

Comment:
It looks like the source table was partitioned by load time, so this would need to be `_PARTITIONTIME`. Since this is metadata where this likely to be only one row per day I'm not sure if we need to partition this at all.

Reply:
Yeah, I don't think it needs to be partitioned at all. I guess I _thought_ I had told it not to partition when I created the table. Are you seeing it partitioned on your end? 

In any event, happy to change this as needed!

Reply:
MM, nope, you're right, it's partitioned. Dingdangit.